bloom.go

```diff
@@ -0,0 +1,230 @@
+package parquet
+
+import (
+	"io"
+	"unsafe"
+
+	"github.com/segmentio/parquet-go/bloom"
+	"github.com/segmentio/parquet-go/deprecated"
+	"github.com/segmentio/parquet-go/encoding"
+	"github.com/segmentio/parquet-go/format"
+)
+
+// BloomFilter is an interface allowing applications to test whether a key
+// exists in a bloom filter.
+type BloomFilter interface {
+	// Implement the io.ReaderAt interface as a mechanism to allow reading the
+	// raw bits of the filter.
+	io.ReaderAt
+	// Returns the size of the bloom filter (in bytes).
+	Size() int64
+	// Tests whether the key is present in the filter.
```

Comment:
Not obvious to me why this would or could return an error.

Reply:
The filters are serialized to storage, the API here doesn't say anything about where they are read from, so it's possible that reading the data failed and may return an error. Any practical data store should probably add a caching layer here so most of the time the filters are in memory, but this is not enforced by the library.

I'll update the documentation to reflect that.